requirements.test.txt

```diff
@@ -0,0 +1,25 @@
+atomicwrites==1.2.1
+attrs==18.2.0
+Automat==0.7.0
+constantly==15.1.0
+coverage==4.5.1
+decorator==4.3.0
+greenlet==0.4.15
+hyperlink==18.0.0
+idna==2.7
+incremental==17.5.0
+more-itertools==4.3.0
+pluggy==0.7.1
+py==1.6.0
+PyHamcrest==1.9.0
+pytest==3.8.0
+pytest-cov==2.6.0
+pytest-twisted==1.8
+-e .[test]
+six==1.11.0
+tox==3.2.1
+Twisted==18.9.0rc1; platform_system != "Windows" and platform_system != "Darwin"
```

Comment:
Are these `platform_system` conditionals necessary? Won't Twisted itself resolve these?

Reply:
No, it does not.  I asked about this and it was pointed out that it can be used for various things without specifying the `windows_platform` extra so they didn't want to force the extra deps.  I don't know offhand which features require it or not, but I thought the default reactor did.

Reply:
Ah, I suppose that makes sense given Twisted's "Swiss army knife"-like flexibility.. Thanks for clarifying.